mappings/net/minecraft/client/gui/MultilineTextField.mapping

```diff
@@ -0,0 +1,164 @@
+CLASS net/minecraft/unmapped/C_krhsyzys net/minecraft/client/gui/MultilineTextField
+	COMMENT A multiline text field with support for basic keyboard shortcuts.
+	COMMENT
+	COMMENT <p>
+	COMMENT This should not be used directly; {@link net.minecraft.client.gui.widget.EditBoxWidget}
+	COMMENT provides the GUI for the edit box.
+	FIELD f_burfhmzf displayLines Ljava/util/List;
+	FIELD f_enhzgdct cursor I
+	FIELD f_fwmsxqxf UNLIMITED_LENGTH I
+		COMMENT A constant denoting that the multiline text field accepts unlimited amount of text.
```

Comment:
```suggestion
		COMMENT A constant denoting that the multiline text field accepts an unlimited amount of text.
```
I guess `the multiline text field` could be shortened to just `the text field` here
```suggestion
		COMMENT A constant denoting that the text field accepts an unlimited amount of text.
```